server/mysterium_api_fake.go

```diff
@@ -8,44 +8,50 @@ import (
 	dto_discovery "github.com/mysterium/node/service_discovery/dto"
 )
 
+// NewClientFake constructs fake API client
 func NewClientFake() *ClientFake {
 	return &ClientFake{
 		proposalsMock: make([]dto_discovery.ServiceProposal, 0),
 	}
 }
 
+// ClientFake is fake client to Mysterium API
 type ClientFake struct {
 	RegisteredIdentity identity.Identity
 	proposalsMock      []dto_discovery.ServiceProposal
 }
 
+// RegisterProposal announces service proposal
 func (client *ClientFake) RegisterProposal(proposal dto_discovery.ServiceProposal, signer identity.Signer) (err error) {
 	client.proposalsMock = append(client.proposalsMock, proposal)
-	log.Info(mysteriumAPILogPrefix, "Fake node registered: ", proposal)
+	log.Info(mysteriumAPILogPrefix, "Fake proposal registered: ", proposal)
 
 	return nil
 }
 
-func (client *ClientFake) RegisterIdentity(newIdentity identity.Identity, signer identity.Signer) (err error) {
-	client.RegisteredIdentity = newIdentity
-	log.Info(mysteriumAPILogPrefix, "Fake newIdentity registered: ", newIdentity)
+// RegisterIdentity announces that new identity was created
+func (client *ClientFake) RegisterIdentity(id identity.Identity, signer identity.Signer) (err error) {
+	client.RegisteredIdentity = id
+	log.Info(mysteriumAPILogPrefix, "Fake newIdentity registered: ", id.Address)
 
 	return nil
 }
 
-func (client *ClientFake) NodeSendStats(nodeKey string, signer identity.Signer) (err error) {
-	log.Info(mysteriumAPILogPrefix, "Node stats sent: ", nodeKey)
+// SendProposalStats heartbeats that service proposal is still active
```

Comment:
This comment is false at the moment - we are passing just `providerID` here and no calculations is done in discovery on whether proposal is active or not. All proposals are active at the memotn. What about this?
```
// SendProposalStats heartbeats that provider is still active
```

Reply:
We dont have unique ID for proposals, yet. But intentions is to ping proposal